index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <header class="toolbar">
+        <p class="logo">My Logo</p>
+        <a class="menu-logo">
+            <div></div>
+            <div></div>
+            <div></div>
+        </a>
+    </header>
+    <section class="headline">
+        <h1 class="greeting">I am Agneta. <br>
+            Glad to see you!
+        </h1>
+        <p class="about">
+             I am a software developer! I can help you build a product,
+             feature or website. Have a look at some of my work and experience! 
+             If you like what you see and have a project you need coded, do not 
+             hesitate to contact me.
+        </p>
+        <br>
+        <h2>LETS CONNECT</h2>
+        <section class="socials">
+            <a><img src="images/Vector.svg" alt="social media"></a>
+            <a><img src="images/Linkedin icon.svg" alt="social media"></a>
+            <a><img src="images/Vector (1).svg" alt="social media"></a>
+            <a><img src="images/Vector (2).svg" alt="social media"></a>
+            <a><img src="images/Vector (3).svg" alt="social media"></a>
+        </section>
```

Comment:
- [ ] According to the requirements. I highly suggest that the list of icons should be structured using a <ul> (unordered list) with each icon wrapped in an <li> (list item) element. Additionally, each icon should have an <a> (anchor) tag with the appropriate "href" attribute to connect to the corresponding social network. Please update the code as follows:

Reply:
list of icons was structured as a list items.